### PR TITLE
Update the PSL_ISO-8859-*.h  include files

### PR DESCRIPTION
**Pull request by PaulWessel**
See #4565 for details.  This PR adds 8859-16 (and 11 but that requires non-standard fonts) and updates those that changed.  Thanks to @uleysky; further changes will require additional changes to how we handle **PS_ENCODING**.  I may add some more ISO plots for testing later but wanted to get this PR in first.

### CODE REVIEW

**Comment by PaulWessel** [comment]
I should have noted it made a tiny difference to one PS original based on ISO-8859-2 so that file has been updated.

File: doc/rst/source/gmt.conf.rst [hunk 1]
```diff
@@ -944,12 +944,12 @@ PostScript Parameters
         files and in command line parameters. This allows GMT to ensure
         that the PostScript output generates the correct characters on the
         plot. Choose from Standard, Standard+, ISOLatin1, ISOLatin1+, and
-        ISO-8859-x (where x is in the ranges [1,10] or [13,15]). See
+        ISO-8859-x (where x is in the ranges [1,11] or [13,15-16]). See
```
Comment (seisman) [inline doc/rst/source/gmt.conf.rst:947]:
```suggestion
        ISO-8859-x (where x is in the ranges 1-11 or 13-16). See
```

**Review by seisman — APPROVED** [review]

File: doc/rst/source/gmt.conf.rst (hunk 1 above)
Comment (PaulWessel) [reply doc/rst/source/gmt.conf.rst:947]: But there is no 14, no?

Comment (seisman) [reply doc/rst/source/gmt.conf.rst:947]: Is this 14?

https://github.com/GenericMappingTools/gmt/blob/master/src/PSL_ISO-8859-14.h

**Comment by PaulWessel** [comment]
Oops...

**Pull request merged**